src/explorer/documentListTreeItem.ts

```diff
@@ -55,6 +55,27 @@ export const formatDocCount = (count: number): string => {
   return `${numeral(count).format('0a')}`.toUpperCase();
 };
 
+function getIconPath():
+    | string
+    | vscode.Uri
+    | { light: string | vscode.Uri; dark: string | vscode.Uri } {
```

Comment:
I know this is from the previous code and occurs in a couple places, but we only ever use the last type signature here, and the other ones could be dropped, right?

Reply:
Good call, I think maybe we can clean these up a bit more in future too. Pushed update.